gcp/table_gcp_kubernetes_cluster.go

```diff
@@ -0,0 +1,423 @@
+package gcp
+
+import (
+	"context"
+	"strings"
+
+	"github.com/turbot/steampipe-plugin-sdk/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/plugin/transform"
+	"google.golang.org/api/container/v1"
+)
+
+//// TABLE DEFINITION
+
+func tableGcpKubernetesCluster(ctx context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "gcp_kubernetes_cluster",
+		Description: "GCP Kubernetes Cluster",
+		List: &plugin.ListConfig{
+			Hydrate: listKubernetesClusters,
+		},
+		Get: &plugin.GetConfig{
+			KeyColumns: plugin.AllColumns([]string{"name", "location"}),
+			Hydrate:    getKubernetesCluster,
+		},
+		Columns: []*plugin.Column{
+			{
+				Name:        "name",
+				Description: "The name of this cluster.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "self_link",
+				Description: "Server-defined URL for the resource.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "location_type",
+				Description: "Location type of the cluster i.e REGIONAL/ZONAL.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.From(gcpKubernetesClusterLocationType),
+			},
+			{
+				Name:        "status",
+				Description: "The current status of this cluster.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "description",
+				Description: "An optional description of this cluster.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "create_time",
+				Description: "The time the cluster was created, in RFC3339 text format.",
+				Type:        proto.ColumnType_TIMESTAMP,
+			},
+			{
+				Name:        "cluster_ipv4_cidr",
+				Description: "The IP address range of the container pods in this cluster, in CIDR notation.",
+				Type:        proto.ColumnType_STRING,
```

Comment:
From description, it seems that the data is a CIDR range.
If so, can we use `ColumnType_CIDR`?